modules/openapi-generator/src/main/resources/typescript-fetch/apis.mustache

```diff
@@ -337,7 +337,7 @@ export class {{classname}} extends runtime.BaseAPI {
         return await response.value();
         {{/returnType}}
         {{^returnType}}
-        await this.{{nickname}}Raw({{#allParams.0}}{ {{#allParams}}{{paramName}}: {{paramName}}{{^-last}}, {{/-last}}{{/allParams}} }{{/allParams.0}}, initOverrides);
+        await this.{{nickname}}Raw({{#allParams.0}}{ {{#allParams}}{{paramName}}: {{paramName}}{{^-last}}, {{/-last}}{{/allParams}} }, {{/allParams.0}}initOverrides);
```

Comment:
this is to fix a bug caused by someone else but I found the fix. The coma was present even if the first parameter was not. My sample pet store exposed this issue and the generated typescript code did not compile correctly.